tests/napi/src/async.rs

```diff
@@ -95,7 +95,7 @@ extern "C" fn test_async_work(
   ));
   let mut baton = unsafe { Box::from_raw(baton_ptr as *mut Baton) };
   baton.task = async_work;
-  Box::into_raw(baton);
+  let _ = Box::into_raw(baton);
```

Comment:
Ditto, what's getting flagged here?

Reply:
I'm getting clippy lint error for unused return value